opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
What does `composable=True` mean to a technique writer?  It sounds like an implementation specific thing about how we generate techniques leaking into on of the main APIs.  If this is not a fundamental part of what a technique is, it should not be in the register API.

- If we want to group techniques by class, we should do `set(map(type, techniques))` or allow register to take class names in addition to prototype instances.
- If we want to test for subclasses of C.E.T., we should use `isinstance()`
- If we want to bias probabilities of selection, we should add something like `Technique.selection_weight = 1.0` -- which would be more general to other methods of generating techniques, and also future classes of techniques that are not evolutionary.